### ISSUE TITLE
Support cosine learning rate scheduling

### ISSUE THREAD

**Issue by iizukak** [blue-oil/blueoil#711]
Adding `cosine_decay` is reasonable, I think.

https://www.tensorflow.org/api_docs/python/tf/compat/v1/train/cosine_decay

**Comment by iizukak** [comment]
closed with #719

**Comment by ed728** [comment]
@iizukak Should this issue be closed then?

**Comment by iizukak** [comment]
@ed728 thanks!!
